lib/src/model.dart

```diff
@@ -13,6 +13,22 @@ import 'pubspec.dart';
 
 part 'model.g.dart';
 
+@JsonSerializable(includeIfNull: false)
+
+/// A processed screenshot contains paths with processed images as stored in GCS.
```

Comment:
```suggestion
/// A processed screenshot contains paths with processed images as stored using `storeResource` as passed to `PackageAnalyzer.inspectPackage`.
```
This is correct, right? This isn't necessarily the path in GCS, this is the path that was passed to `storeResource`.

What `storeResource` does with the path name is irrelevant to pana. When pub.dev calls pana it probably chooses to store the result in some sub-folder on GCS. But that detail does not concern pana.

Reply:
Done.